functions/test/util/admin.ts

```diff
@@ -0,0 +1,17 @@
+import * as functions from 'firebase-functions';
```

Comment:
maybe we could name this file `config.ts` or something like that, since it's not just the admin